pontoon/base/migrations/0012_auto_20150804_0859.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+def remove_unused_locales(apps, schema_editor):
+    Locale = apps.get_model('base', 'Locale')
+
+    for unused_locale in UNUSED_LOCALES:
```

Comment:
Wouldn't `Locale.objects.filter(code__in=UNUSED_LOCALES).delete()` work as well? Not that it matters really, I'm more curious if I'm missing something.